packages/api/src/brevo-module.ts

```diff
@@ -30,10 +29,10 @@ export class BrevoModule {
         }
     }
     static register(config: BrevoModuleConfig): DynamicModule {
-        const TargetGroup = TargetGroupEntityFactory.create({
-            Scope: config.emailCampaigns.Scope,
-            BrevoFilterAttributes: config.brevo.BrevoContactFilterAttributes,
-        });
+        // const TargetGroup = createTargetGroupEntity({
+        //     Scope: config.emailCampaigns.Scope,
+        //     BrevoFilterAttributes: config.brevo.BrevoContactFilterAttributes,
+        // });
```

Comment:
Can be removed

Reply:
38c4b8a2234b2e6ed98ac735a43f32269f5f615c